scapy/config.py

```diff
@@ -513,7 +514,7 @@ class Conf(ConfClass):
                    'llmnr', 'lltd', 'mgcp', 'mobileip', 'netbios',
                    'netflow', 'ntp', 'ppp', 'pptp', 'radius', 'rip',
                    'rtp', 'sctp', 'sixlowpan', 'skinny', 'smb', 'snmp',
-                   'tftp', 'vrrp', 'vxlan', 'x509', 'zigbee']
+                   'tftp', 'usb', 'vrrp', 'vxlan', 'x509', 'zigbee']
```

Comment:
I think that it will be better to remove it from the list and explicitly load it with `load_layers("usb")`.